core/agent.py

```
@@ -1,172 +1,250 @@
-                cli.display_action("content", response["content"])
-                self.state.add_message("assistant", response["content"])
-
-            user_input = cli.display_question("请继续输入你的问题或任务，或输入'exit'退出：")
-            if user_input.lower() in ("exit", "quit", "退出"):
-                self.state.update_spec({"status": "completed"})
-                cli.display_result("用户退出", True)
+            
+            # 调用 LLM
+            try:
+                assistant_response = self.llm.chat(self.conversation)
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)